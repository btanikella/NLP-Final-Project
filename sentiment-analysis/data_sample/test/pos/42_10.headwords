one
rich
draw
taken
executed
wonderful
want
