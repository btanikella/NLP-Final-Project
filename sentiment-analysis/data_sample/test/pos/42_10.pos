DT VBZ CD IN DT NNS IN PRP RB VBP RB VB TO VB
DT NNS VBP JJ IN DT RB VBN NN
JJ NNS  NN CC NNS VBP PRP IN CC VB DT NN IN DT NNS WDT VBD IN DT VBG NN IN DT NNS IN DT NN IN DT JJ NNS NN CC DT NNP NNP
DT JJ NN VBZ VBN RP IN DT JJ  JJ NN WDT VBZ DT NN CC DT VBP NNS VBN IN PRP$ NN MD RB VB DT VBD DT VBN IN NN NN
PRP VBD RB RB VBN
NNP NNP NNP VBZ JJ IN NNP CC PRP VBZ RB JJ NN TO DT NN
PRP VBP TO VB RB RB IN PRP$ NN VBD DT NN RB JJ PRP VBD DT NN
