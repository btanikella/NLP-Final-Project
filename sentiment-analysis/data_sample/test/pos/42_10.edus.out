Sentence #1 (15 tokens):
This is one of those movies that you just don't want to end.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=9 CharacterOffsetEnd=11 PartOfSpeech=VBZ] [Text=one CharacterOffsetBegin=12 CharacterOffsetEnd=15 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=16 CharacterOffsetEnd=18 PartOfSpeech=IN] [Text=those CharacterOffsetBegin=19 CharacterOffsetEnd=24 PartOfSpeech=DT] [Text=movies CharacterOffsetBegin=25 CharacterOffsetEnd=31 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=32 CharacterOffsetEnd=36 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=37 CharacterOffsetEnd=40 PartOfSpeech=PRP] [Text=just CharacterOffsetBegin=41 CharacterOffsetEnd=45 PartOfSpeech=RB] [Text=do CharacterOffsetBegin=46 CharacterOffsetEnd=48 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=48 CharacterOffsetEnd=51 PartOfSpeech=RB] [Text=want CharacterOffsetBegin=52 CharacterOffsetEnd=56 PartOfSpeech=VB] [Text=to CharacterOffsetBegin=57 CharacterOffsetEnd=59 PartOfSpeech=TO] [Text=end CharacterOffsetBegin=60 CharacterOffsetEnd=63 PartOfSpeech=VB] [Text=. CharacterOffsetBegin=63 CharacterOffsetEnd=64 PartOfSpeech=.] 
root(ROOT-0, one-3)
nsubj(one-3, This-1)
cop(one-3, is-2)
prep(one-3, of-4)
det(movies-6, those-5)
pobj(of-4, movies-6)
dobj(want-12, that-7)
nsubj(want-12, you-8)
advmod(want-12, just-9)
aux(want-12, do-10)
neg(want-12, n't-11)
rcmod(one-3, want-12)
aux(end-14, to-13)
xcomp(want-12, end-14)
punct(one-3, .-15)

Sentence #2 (10 tokens):
The characters are rich like a well woven tapestry.
[Text=The CharacterOffsetBegin=69 CharacterOffsetEnd=72 PartOfSpeech=DT] [Text=characters CharacterOffsetBegin=73 CharacterOffsetEnd=83 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=84 CharacterOffsetEnd=87 PartOfSpeech=VBP] [Text=rich CharacterOffsetBegin=88 CharacterOffsetEnd=92 PartOfSpeech=JJ] [Text=like CharacterOffsetBegin=93 CharacterOffsetEnd=97 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=98 CharacterOffsetEnd=99 PartOfSpeech=DT] [Text=well CharacterOffsetBegin=100 CharacterOffsetEnd=104 PartOfSpeech=RB] [Text=woven CharacterOffsetBegin=105 CharacterOffsetEnd=110 PartOfSpeech=VBN] [Text=tapestry CharacterOffsetBegin=111 CharacterOffsetEnd=119 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=119 CharacterOffsetEnd=120 PartOfSpeech=.] 
root(ROOT-0, rich-4)
det(characters-2, The-1)
nsubj(rich-4, characters-2)
cop(rich-4, are-3)
prep(rich-4, like-5)
det(tapestry-9, a-6)
advmod(woven-8, well-7)
amod(tapestry-9, woven-8)
pobj(like-5, tapestry-9)
punct(rich-4, .-10)

Sentence #3 (38 tokens):
Colorful costumes, music and characters draw you in and tell a tale of the people that lived in a boarding house over the decades around the time of the civil rights movement and the Vietnam War.
[Text=Colorful CharacterOffsetBegin=125 CharacterOffsetEnd=133 PartOfSpeech=JJ] [Text=costumes CharacterOffsetBegin=134 CharacterOffsetEnd=142 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=142 CharacterOffsetEnd=143 PartOfSpeech=,] [Text=music CharacterOffsetBegin=144 CharacterOffsetEnd=149 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=150 CharacterOffsetEnd=153 PartOfSpeech=CC] [Text=characters CharacterOffsetBegin=154 CharacterOffsetEnd=164 PartOfSpeech=NNS] [Text=draw CharacterOffsetBegin=165 CharacterOffsetEnd=169 PartOfSpeech=VBP] [Text=you CharacterOffsetBegin=170 CharacterOffsetEnd=173 PartOfSpeech=PRP] [Text=in CharacterOffsetBegin=174 CharacterOffsetEnd=176 PartOfSpeech=IN] [Text=and CharacterOffsetBegin=177 CharacterOffsetEnd=180 PartOfSpeech=CC] [Text=tell CharacterOffsetBegin=181 CharacterOffsetEnd=185 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=186 CharacterOffsetEnd=187 PartOfSpeech=DT] [Text=tale CharacterOffsetBegin=188 CharacterOffsetEnd=192 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=193 CharacterOffsetEnd=195 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=196 CharacterOffsetEnd=199 PartOfSpeech=DT] [Text=people CharacterOffsetBegin=200 CharacterOffsetEnd=206 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=207 CharacterOffsetEnd=211 PartOfSpeech=WDT] [Text=lived CharacterOffsetBegin=212 CharacterOffsetEnd=217 PartOfSpeech=VBD] [Text=in CharacterOffsetBegin=218 CharacterOffsetEnd=220 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=221 CharacterOffsetEnd=222 PartOfSpeech=DT] [Text=boarding CharacterOffsetBegin=223 CharacterOffsetEnd=231 PartOfSpeech=VBG] [Text=house CharacterOffsetBegin=232 CharacterOffsetEnd=237 PartOfSpeech=NN] [Text=over CharacterOffsetBegin=238 CharacterOffsetEnd=242 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=243 CharacterOffsetEnd=246 PartOfSpeech=DT] [Text=decades CharacterOffsetBegin=247 CharacterOffsetEnd=254 PartOfSpeech=NNS] [Text=around CharacterOffsetBegin=255 CharacterOffsetEnd=261 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=262 CharacterOffsetEnd=265 PartOfSpeech=DT] [Text=time CharacterOffsetBegin=266 CharacterOffsetEnd=270 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=271 CharacterOffsetEnd=273 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=DT] [Text=civil CharacterOffsetBegin=278 CharacterOffsetEnd=283 PartOfSpeech=JJ] [Text=rights CharacterOffsetBegin=284 CharacterOffsetEnd=290 PartOfSpeech=NNS] [Text=movement CharacterOffsetBegin=291 CharacterOffsetEnd=299 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=300 CharacterOffsetEnd=303 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=304 CharacterOffsetEnd=307 PartOfSpeech=DT] [Text=Vietnam CharacterOffsetBegin=308 CharacterOffsetEnd=315 PartOfSpeech=NNP] [Text=War CharacterOffsetBegin=316 CharacterOffsetEnd=319 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=319 CharacterOffsetEnd=320 PartOfSpeech=.] 
root(ROOT-0, draw-7)
amod(costumes-2, Colorful-1)
nsubj(draw-7, costumes-2)
punct(costumes-2, ,-3)
conj(costumes-2, music-4)
cc(costumes-2, and-5)
conj(costumes-2, characters-6)
dobj(draw-7, you-8)
prt(draw-7, in-9)
cc(draw-7, and-10)
conj(draw-7, tell-11)
det(tale-13, a-12)
dobj(tell-11, tale-13)
prep(tale-13, of-14)
det(people-16, the-15)
pobj(of-14, people-16)
nsubj(lived-18, that-17)
rcmod(people-16, lived-18)
prep(lived-18, in-19)
det(house-22, a-20)
amod(house-22, boarding-21)
pobj(in-19, house-22)
prep(lived-18, over-23)
det(decades-25, the-24)
pobj(over-23, decades-25)
prep(lived-18, around-26)
det(time-28, the-27)
pobj(around-26, time-28)
prep(time-28, of-29)
det(movement-33, the-30)
amod(movement-33, civil-31)
nn(movement-33, rights-32)
pobj(of-29, movement-33)
cc(movement-33, and-34)
det(War-37, the-35)
nn(War-37, Vietnam-36)
conj(movement-33, War-37)
punct(draw-7, .-38)

Sentence #4 (35 tokens):
A young man is taken in by a dynamic, big-hearted woman that runs the house and these are stories based on his experiences.I couldn't believe this was a made for television film.
[Text=A CharacterOffsetBegin=325 CharacterOffsetEnd=326 PartOfSpeech=DT] [Text=young CharacterOffsetBegin=327 CharacterOffsetEnd=332 PartOfSpeech=JJ] [Text=man CharacterOffsetBegin=333 CharacterOffsetEnd=336 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=337 CharacterOffsetEnd=339 PartOfSpeech=VBZ] [Text=taken CharacterOffsetBegin=340 CharacterOffsetEnd=345 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=346 CharacterOffsetEnd=348 PartOfSpeech=RP] [Text=by CharacterOffsetBegin=349 CharacterOffsetEnd=351 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=352 CharacterOffsetEnd=353 PartOfSpeech=DT] [Text=dynamic CharacterOffsetBegin=354 CharacterOffsetEnd=361 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=361 CharacterOffsetEnd=362 PartOfSpeech=,] [Text=big-hearted CharacterOffsetBegin=363 CharacterOffsetEnd=374 PartOfSpeech=JJ] [Text=woman CharacterOffsetBegin=375 CharacterOffsetEnd=380 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=381 CharacterOffsetEnd=385 PartOfSpeech=WDT] [Text=runs CharacterOffsetBegin=386 CharacterOffsetEnd=390 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=391 CharacterOffsetEnd=394 PartOfSpeech=DT] [Text=house CharacterOffsetBegin=395 CharacterOffsetEnd=400 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=401 CharacterOffsetEnd=404 PartOfSpeech=CC] [Text=these CharacterOffsetBegin=405 CharacterOffsetEnd=410 PartOfSpeech=DT] [Text=are CharacterOffsetBegin=411 CharacterOffsetEnd=414 PartOfSpeech=VBP] [Text=stories CharacterOffsetBegin=415 CharacterOffsetEnd=422 PartOfSpeech=NNS] [Text=based CharacterOffsetBegin=423 CharacterOffsetEnd=428 PartOfSpeech=VBN] [Text=on CharacterOffsetBegin=429 CharacterOffsetEnd=431 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=432 CharacterOffsetEnd=435 PartOfSpeech=PRP$] [Text=experiences.I CharacterOffsetBegin=436 CharacterOffsetEnd=449 PartOfSpeech=NN] [Text=could CharacterOffsetBegin=450 CharacterOffsetEnd=455 PartOfSpeech=MD] [Text=n't CharacterOffsetBegin=455 CharacterOffsetEnd=458 PartOfSpeech=RB] [Text=believe CharacterOffsetBegin=459 CharacterOffsetEnd=466 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=467 CharacterOffsetEnd=471 PartOfSpeech=DT] [Text=was CharacterOffsetBegin=472 CharacterOffsetEnd=475 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=476 CharacterOffsetEnd=477 PartOfSpeech=DT] [Text=made CharacterOffsetBegin=478 CharacterOffsetEnd=482 PartOfSpeech=VBN] [Text=for CharacterOffsetBegin=483 CharacterOffsetEnd=486 PartOfSpeech=IN] [Text=television CharacterOffsetBegin=487 CharacterOffsetEnd=497 PartOfSpeech=NN] [Text=film CharacterOffsetBegin=498 CharacterOffsetEnd=502 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=502 CharacterOffsetEnd=503 PartOfSpeech=.] 
root(ROOT-0, taken-5)
det(man-3, A-1)
amod(man-3, young-2)
nsubjpass(taken-5, man-3)
auxpass(taken-5, is-4)
prt(taken-5, in-6)
prep(taken-5, by-7)
det(woman-12, a-8)
amod(woman-12, dynamic-9)
punct(woman-12, ,-10)
amod(woman-12, big-hearted-11)
pobj(by-7, woman-12)
nsubj(runs-14, that-13)
rcmod(woman-12, runs-14)
det(house-16, the-15)
dobj(runs-14, house-16)
cc(runs-14, and-17)
nsubj(stories-20, these-18)
cop(stories-20, are-19)
conj(runs-14, stories-20)
partmod(stories-20, based-21)
prep(based-21, on-22)
poss(experiences.I-24, his-23)
pobj(on-22, experiences.I-24)
aux(believe-27, could-25)
neg(believe-27, n't-26)
xcomp(taken-5, believe-27)
dobj(believe-27, this-28)
partmod(this-28, was-29)
dep(was-29, a-30)
partmod(a-30, made-31)
prep(made-31, for-32)
nn(film-34, television-33)
pobj(for-32, film-34)
punct(taken-5, .-35)

Sentence #5 (6 tokens):
It was so well executed.
[Text=It CharacterOffsetBegin=508 CharacterOffsetEnd=510 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=511 CharacterOffsetEnd=514 PartOfSpeech=VBD] [Text=so CharacterOffsetBegin=515 CharacterOffsetEnd=517 PartOfSpeech=RB] [Text=well CharacterOffsetBegin=518 CharacterOffsetEnd=522 PartOfSpeech=RB] [Text=executed CharacterOffsetBegin=523 CharacterOffsetEnd=531 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=531 CharacterOffsetEnd=532 PartOfSpeech=.] 
root(ROOT-0, executed-5)
nsubjpass(executed-5, It-1)
auxpass(executed-5, was-2)
advmod(well-4, so-3)
advmod(executed-5, well-4)
punct(executed-5, .-6)

Sentence #6 (17 tokens):
S. Epatha Merkerson is wonderful as Nanny and she brings so much life to this role.
[Text=S. CharacterOffsetBegin=537 CharacterOffsetEnd=539 PartOfSpeech=NNP] [Text=Epatha CharacterOffsetBegin=540 CharacterOffsetEnd=546 PartOfSpeech=NNP] [Text=Merkerson CharacterOffsetBegin=547 CharacterOffsetEnd=556 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=557 CharacterOffsetEnd=559 PartOfSpeech=VBZ] [Text=wonderful CharacterOffsetBegin=560 CharacterOffsetEnd=569 PartOfSpeech=JJ] [Text=as CharacterOffsetBegin=570 CharacterOffsetEnd=572 PartOfSpeech=IN] [Text=Nanny CharacterOffsetBegin=573 CharacterOffsetEnd=578 PartOfSpeech=NNP] [Text=and CharacterOffsetBegin=579 CharacterOffsetEnd=582 PartOfSpeech=CC] [Text=she CharacterOffsetBegin=583 CharacterOffsetEnd=586 PartOfSpeech=PRP] [Text=brings CharacterOffsetBegin=587 CharacterOffsetEnd=593 PartOfSpeech=VBZ] [Text=so CharacterOffsetBegin=594 CharacterOffsetEnd=596 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=597 CharacterOffsetEnd=601 PartOfSpeech=JJ] [Text=life CharacterOffsetBegin=602 CharacterOffsetEnd=606 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=607 CharacterOffsetEnd=609 PartOfSpeech=TO] [Text=this CharacterOffsetBegin=610 CharacterOffsetEnd=614 PartOfSpeech=DT] [Text=role CharacterOffsetBegin=615 CharacterOffsetEnd=619 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=619 CharacterOffsetEnd=620 PartOfSpeech=.] 
root(ROOT-0, wonderful-5)
nn(Merkerson-3, S.-1)
nn(Merkerson-3, Epatha-2)
nsubj(wonderful-5, Merkerson-3)
cop(wonderful-5, is-4)
prep(wonderful-5, as-6)
pobj(as-6, Nanny-7)
cc(wonderful-5, and-8)
nsubj(brings-10, she-9)
conj(wonderful-5, brings-10)
advmod(much-12, so-11)
amod(life-13, much-12)
dobj(brings-10, life-13)
prep(brings-10, to-14)
det(role-16, this-15)
pobj(to-14, role-16)
punct(wonderful-5, .-17)

Sentence #7 (19 tokens):
You want to be right there amongst her boarders.I enjoyed this film so much I bought the DVD.
[Text=You CharacterOffsetBegin=625 CharacterOffsetEnd=628 PartOfSpeech=PRP] [Text=want CharacterOffsetBegin=629 CharacterOffsetEnd=633 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=634 CharacterOffsetEnd=636 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=637 CharacterOffsetEnd=639 PartOfSpeech=VB] [Text=right CharacterOffsetBegin=640 CharacterOffsetEnd=645 PartOfSpeech=RB] [Text=there CharacterOffsetBegin=646 CharacterOffsetEnd=651 PartOfSpeech=RB] [Text=amongst CharacterOffsetBegin=652 CharacterOffsetEnd=659 PartOfSpeech=IN] [Text=her CharacterOffsetBegin=660 CharacterOffsetEnd=663 PartOfSpeech=PRP$] [Text=boarders.I CharacterOffsetBegin=664 CharacterOffsetEnd=674 PartOfSpeech=NN] [Text=enjoyed CharacterOffsetBegin=675 CharacterOffsetEnd=682 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=683 CharacterOffsetEnd=687 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=688 CharacterOffsetEnd=692 PartOfSpeech=NN] [Text=so CharacterOffsetBegin=693 CharacterOffsetEnd=695 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=696 CharacterOffsetEnd=700 PartOfSpeech=JJ] [Text=I CharacterOffsetBegin=701 CharacterOffsetEnd=702 PartOfSpeech=PRP] [Text=bought CharacterOffsetBegin=703 CharacterOffsetEnd=709 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=710 CharacterOffsetEnd=713 PartOfSpeech=DT] [Text=DVD CharacterOffsetBegin=714 CharacterOffsetEnd=717 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=717 CharacterOffsetEnd=718 PartOfSpeech=.] 
root(ROOT-0, want-2)
nsubj(want-2, You-1)
aux(right-5, to-3)
cop(right-5, be-4)
xcomp(want-2, right-5)
advmod(enjoyed-10, there-6)
mark(enjoyed-10, amongst-7)
poss(boarders.I-9, her-8)
nsubj(enjoyed-10, boarders.I-9)
ccomp(right-5, enjoyed-10)
det(film-12, this-11)
dobj(enjoyed-10, film-12)
advmod(much-14, so-13)
dobj(enjoyed-10, much-14)
nsubj(bought-16, I-15)
rcmod(much-14, bought-16)
det(DVD-18, the-17)
dobj(bought-16, DVD-18)
punct(want-2, .-19)

